build.gradle.kts

```diff
@@ -21,6 +22,8 @@ dependencies {
     compile("com.google.guava:guava:27.1-jre")
     compile("org.bouncycastle:bcprov-jdk15on:1.61")
     compile("org.bouncycastle:bcpkix-jdk15on:1.61")
+    compile("com.github.multiformats:java-multiaddr:v1.3.1")
```

Comment:
Where is this actually used? I think I missed it.

Reply:
`Cid` class: https://github.com/raulk/jvm-libp2p-minimal/blob/b60fe22ec05dab0b2b495e6bb0fbe66c0578837b/src/main/kotlin/io/libp2p/core/multiformats/Protocol.kt#L70
`Base32` class: https://github.com/raulk/jvm-libp2p-minimal/blob/b60fe22ec05dab0b2b495e6bb0fbe66c0578837b/src/main/kotlin/io/libp2p/core/multiformats/Protocol.kt#L83

@raulk We can exclude IPFS & ONION protocols for now and remove that dependency